src/neuroconv/datainterfaces/ophys/scanimage/scanimageimaginginterfaces.py

```diff
@@ -0,0 +1,687 @@
+import datetime
+import json
+from pathlib import Path
+from typing import Optional
+
+from dateutil.parser import parse as dateparse
+
+from ..baseimagingextractorinterface import BaseImagingExtractorInterface
+from ....utils import FilePathType, FolderPathType
+
+
+class ScanImageImagingInterface(BaseImagingExtractorInterface):
+    """Interface for ScanImage TIFF files."""
```

Comment:
I think it could be useful to give a quick description of the routing path here in the docstring.

Reply:
Thanks, how about this: https://github.com/catalystneuro/neuroconv/pull/809/commits/20ee52eb7064fb48ddfb9c07b9a15f07372fc0c3